components/skeleton/skeleton-mixin.js

```diff
@@ -131,7 +138,7 @@ export const skeletonStyles = css`
 	}
 `;
 
-export const SkeletonMixin = superclass => class extends superclass {
+export const SkeletonMixin = superclass => class extends RtlMixin(superclass) {
```

Comment:
Alright this should fix the initial issues. I was testing it in the demo container which puts `dir="rtl"` right on the element. Without the `RtlMixin`, it wasn't picking up a `dir="rtl"` on the document.

Since having someone use `SkeletonMixin` AND `RtlMixin` -- even within our own components -- is quite common, I've added the de-duper to `RtlMixin`.